pkg/tunnel/util_test.go

```diff
@@ -0,0 +1,324 @@
+package tunnel
+
+import (
+	"fmt"
+	"github.com/libp2p/go-libp2p-core/peer"
```

Comment:
This is not the import order recommended by go, please import in the following order:
1. built-in packages
2. third-party packages
3. this project packages